providers/ethereum-provider/package.json

```diff
@@ -39,15 +39,18 @@
     "@walletconnect/jsonrpc-provider": "^1.0.6",
     "@walletconnect/jsonrpc-types": "^1.0.2",
     "@walletconnect/jsonrpc-utils": "^1.0.4",
+    "@web3modal/standalone": "^2.0.0-rc.3",
```

Comment:
Could use stable here instead of the `rc` if it it's just a drop-in? If it requires more changes better in a follow-up PR 👍